src/event_form/components/calendar/base-calendar.controller.js

```diff
@@ -54,27 +58,20 @@ function BaseCalendarController(calendar, $scope) {
     calendar.weeklyRecurring = isTypeWeeklyRecurring(calendarType);
 
     if (calendarType === 'single' && _.isEmpty(calendar.timeSpans)) {
-      initTimeSpans();
+      calendar.showBootstrapTimeSpans = true;
     }
   }
 
-  function initTimeSpans() {
-    calendar.timeSpans = [
-      {
-        allDay: true,
-        start: moment().startOf('hour').add(1, 'h').toDate(),
-        end: moment().startOf('hour').add(4, 'h').toDate()
-      }
-    ];
-  }
-
   function createTimeSpan() {
     if (_.isEmpty(calendar.timeSpans)) {
-      initTimeSpans();
+      calendar.showBootstrapTimeSpans = true;
```

Comment:
showBootstrapTimeSpans verwees naar de suggestie van de nieuwe kalender maar vervangen we nu **chosenFirstStartDate** en afgeleiden. 'First' staat in de variabelenaam omdat er geen verwarring zou zijn van andere startDates in de timespans.